src/Bridges/lazybridgeoptimizer.jl

```diff
@@ -105,9 +105,9 @@ function supports_bridging_constraint(b::LazyBridgeOptimizer, F::Type{<:MOI.Abst
 end
 function bridge_type(b::LazyBridgeOptimizer{BT}, F::Type{<:MOI.AbstractFunction}, S::Type{<:MOI.AbstractSet}) where BT
     update_constraint!(b, F, S)
-    BridgeType = get(b.best, (F, S), nothing)
-    if BridgeType === nothing
+    bridge_type = get(b.best, (F, S), nothing)
```

Comment:
I'm assuming this doesn't cause an issue with having the same name as the function itself?